src/ext/porcelain.lisp

```diff
@@ -0,0 +1,345 @@
+
```

Comment:
I know it is trivial, but it would be great if you could remove this blank line.